resource-types/s3-object/awscommunity-s3-object.json

```diff
@@ -51,7 +51,7 @@
         "Tags": {
             "description": "An array of key-value pairs to apply to this resource.",
             "type": "array",
-            "uniqueItems": true,
+            "uniqueItems": false,
```

Comment:
interesting, is this because S3 tags don't have to be unique?

Reply:
Yes it's a slight fix, but I can revert if wanted.